internal/controller/storagemgr/storagedriver/storagehandler_test.go

```diff
@@ -59,7 +59,7 @@ func TestNewCommandValue(t *testing.T) {
 		Type          models.ValueType
 		ErrorExpected bool
 	}{
-		{"Test A Binary", []byte{1, 0, 0, 1}, models.Binary, false},
+		{"Test A Binary", string("AA=="), models.String, false},
```

Comment:
I think it would be better to leave this line as the old one.

Reply:
This was done as earlier all Binary data was being converted to Base64 thus this was necessary. Post adding a check I think this would not be necessary. I will remove it if check passes.

Reply:
Post change, I reverted this.